nva/nvafakebios.c

```diff
@@ -226,14 +231,19 @@ int main(int argc, char **argv) {
 	}
 
 	/* Arguments parsing */
-	while ((c = getopt (argc, argv, "hc:e:E:w:W:l:L:")) != -1)
+	while ((c = getopt (argc, argv, "hc:m:e:E:w:W:l:L:")) != -1)
 		switch (c) {
 			case 'h':
 				usage(0);
 				break;
 			case 'c':
 				sscanf(optarg, "%d", &cnum);
 				break;
+			case 'm':
+				sscanf(optarg, "%d", &vram_size);
+				/* Calculate the physical address for a 2*64 kB chunk on the end of the vram */
+				vbios_vram = ((vram_size * 16) - 2) << 16;
```

Comment:
I'd prefer converting to kb and do the math more explicit. Like: vbios_vram_kb = ((vram_size * 1024) - 128) << 10;

Reply:
ohh the result is actually bytes not kb